lib/hooks/use-pinheader.tsx

```diff
@@ -0,0 +1,8 @@
+import type { PinHeaderProps } from "@tscircuit/props"
+import { createUseComponent } from "./create-use-component"
+
+export const usePinHeader = (pinLabels: string[]) =>
+  createUseComponent(
+    (props: PinHeaderProps) => <pinheader pinLabels={pinLabels} {...props} />,
```

Comment:
Needs to have name prop as first argument, second argument should be { pinLabels } (an options style object). Use the pinlabelspec or PinHeaderProps[“pinLabels”] to accept all forms of pinlabels including object variants

Reply:
Done the changes 👍🏻